synchronizer/connectors/jira.py

```diff
@@ -62,6 +63,22 @@ def export_worklogs(self, worklogs):
                 print(err)
                 raise self.ExportException(i)
 
+    def validate_issue(self, issue_id):
+        """
+        Check if issue exists in JIRA.
+
+        :param str issue_id: Issue id of task in target source
+        :rtype: bool
+        """
+        try:
+            self._get(

```

Comment:
Let's write this in single line (there is no need to split this method call because it too short).

It's minor fix but current view is harder to read